seed-job/Jenkinsfile

```diff
@@ -0,0 +1,5 @@
+monorepoSeedPipeline {
+	platform = ['ubuntu']
+	organizationName = 'Symbol'
+	gitHubId = 'Symbol-Github-app'
+}
```

Comment:
where will this show up? under https://jenkins.symboldev.com/job/Symbol/job/generated/job/product/